src/pages/userPage/Blanks/UserAchievements/ListOfAchievementsModal.tsx

```diff
@@ -55,12 +53,10 @@ const ListOfAchievementsModal = (props: Props) => {
   };
 
   const deleteFIle = async (documentId: number,  userId: string, fileName: string) => {
-    await removeAchievementDocument(documentId ,userId);
+    await removeAchievementDocument(documentId, userId);
```

Comment:
You should add exception handling in case the document cannot be removed.

Reply:
But why it couldn't be removed? If user doesn't have rights to delete the file he won't be able to see the delete button